### PR TITLE
[in progress] Glyph tests

### LINKED ISSUE #401: Tests for ability to render special characters
**Is your feature request related to a problem? Please describe.**
We want to add some tests to check that we can render a list of special characters in a Headline, SubHeading and text (paragraph). The list of special characters is here https://confluence.dev.bbc.co.uk/display/NEWSART/Text+-+Glyphs.

We now have a test article containing all of the characters: `cev804m5n0vo`.

**Describe the solution you'd like**
~This should be in the form of cypress tests. Current consensus is leaning towards an initial manual check, opening tickets to fix as needed, & enforcing long-term via screenshot tests.~

Update: Following the creation of #1370 we have agreed that we can MVP a manual check for the Testers to perform which should be added to the Articles Test Steps. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Manually test that all of the special characters display as expected across numerous devices/browsers. We can implement screenshot tests once these has been enabled.

**Additional context**
n/a

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #401

Workable in theory, but in practice the test JSON data is not available within the E2E test suite. Not sure on next steps - either we:

1) add/change prod data to include special characters
2) allow testing of test urls in the E2E test suite
3) write the tests outside of the E2E test suite

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: data/test/news/articles/c0000000017o.json [hunk 1]
```diff
@@ -188,12 +188,12 @@
                     {
                       "type": "paragraph",
                       "model": {
-                        "text": "Maecenas varius diam in mi dignissim, eget maximus lacus commodo",
+                        "text": "Maecenas $,%,+,-,=,_,£,¥,©,°,²,³,¹,×,ẞ,ß,à,á,â,æ,ç,è,é,ê,î,ô,÷,û,Ŵ,ŵ,Ŷ,ŷ,⁰,⁴,⁵,⁶,⁷,⁸,⁹,₀,₁,₂,₃,₄,₅,₆,₇,₈,₉,€ diam in mi dignissim, eget maximus lacus commodo",
```
Comment (dr3) [inline data/test/news/articles/c0000000017o.json:191]: Just double checking, do the characters come from aries like this, or are they encoded in any way?

Comment (ChrisBAshton) [reply data/test/news/articles/c0000000017o.json:191]: Good question 👍don't know! Will ask in Slack

Comment (dr3) [reply data/test/news/articles/c0000000017o.json:191]: I tried to make test asset cww18elgprxo on test to check, but it never surfaced 🙈

Comment (dr3) [reply data/test/news/articles/c0000000017o.json:191]: Cameron on slack confirmed they are not encoded, so all good

**Comment by ChrisBAshton** [comment]
Was able to test with production data by `export NODE_ENV='production'`, followed by `npm run build` and `npm start`.

Tried removing `<meta charSet="utf-8" />`, and also actively setting it to `ascii`/`utf-16`/`ansi` and could _not get the page to break_ on any of the following characters: `κόσμε “😋 $,%,+,-,=,_,£,¥,©,°,²,³,¹,×,ẞ,ß,à,á,â,æ,ç,è,é,ê,î,ô,÷,û,Ŵ,ŵ,Ŷ,ŷ,⁰,⁴,⁵,⁶,⁷,⁸,⁹,₀,₁,₂,₃,₄,₅,₆,₇,₈,₉,€`

So, I wasn't able to test whether or not a simple `containsText()` test would be good enough.

Abandoning for now, will share my (lack of) findings with the test team.

**Comment by greenc05** [comment]
@ChrisBAshton not sure if this is relevant yet but some of the glyphs need to be output in a certain way, such as superscript and subscript, they need to be a `<sup>` or `<sub>` HTML element rather than a HTML character code such as `&sup1;`

**Comment by ChrisBAshton** [comment]
Very relevant - thanks @greenc05. Will expand on #401.

**Pull request closed without merging**